Sources/PointFree/Blog/BlogPosts/BlogPost0001_Announcement.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+
+let post0001_welcome = BlogPost(
```

Comment:
do we wanna keep things simple and do the same as we do with `ep1`, etc.?

Reply:
I was thinking of even changing `ep1` and friends to something more complicit so that it'd be nicer to link to particular episodes.